Mathlib/Data/Option/Basic.lean

```diff
@@ -438,4 +438,28 @@ theorem elim_comp₂ (h : α → β → γ) {f : γ → α} {x : α} {g : γ →
 theorem elim_apply {f : γ → α → β} {x : α → β} {i : Option γ} {y : α} :
     i.elim x f y = i.elim (x y) fun j => f j y := by rw [elim_comp fun f : α → β => f y]
 
+@[simp]
+lemma bnot_isSome (a : Option α) : a.isSome.not = a.isNone := by
```

Comment:
Same questions about `Bool.not` in this file

Reply:
Ok, I figured out what I was doing wrong, this is now fixed